packages/sys-apps/cpuid/cpuid-20201006.exheres-0

```diff
@@ -17,12 +17,19 @@ MYOPTIONS="parts: binaries documentation"
 
 DEPENDENCIES=""
 
-BUGS_TO="virkony@gmail.com cpuid@etallen.com"
```

Comment:
Removed from recommended metadata in https://gitlab.exherbo.org/exherbo-misc/docs/-/commit/8886e87da9dfd3818892e334f2c96d83d1d6fb7d